web/src/main/java/org/fao/geonet/monitor/link/LinkInfo.java

```diff
@@ -0,0 +1,91 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LinkInfo {
```

Comment:
Have you read the design document? It follows it one to one.

Only one function here actually has logic.